src/main/java/Events.java

```diff
@@ -0,0 +1,35 @@
+public class Events extends Task{
+
+    protected String from;
+    protected String to;
+
+    public Events(String description){
+        super(description, "E");
+    }
+
+    public String getFrom() {
+        return from;
+    }
+
+    public void setFrom(String from) {
+        this.from = from;
+    }
+
+    public String getTo() {
+        return to;
+    }
+
+    public void setTo(String to) {
+        this.to = to;
+    }
```

Comment:
Should these methods be collapsed into the constructor, since they are not used outside of it?